administrator/components/com_cache/src/View/Cache/HtmlView.php

```diff
@@ -125,8 +125,8 @@ protected function addToolbar(): void
 		// Get the toolbar object instance
 		$toolbar = Toolbar::getInstance('toolbar');
 
-		ToolbarHelper::custom('delete', 'delete.png', 'delete_f2.png', 'JTOOLBAR_DELETE', true);
-		ToolbarHelper::custom('deleteAll', 'remove.png', 'delete_f2.png', 'JTOOLBAR_DELETE_ALL', false);
+		ToolbarHelper::custom('delete', 'delete.png', '', 'JTOOLBAR_DELETE', true);
```

Comment:
```suggestion
		ToolbarHelper::custom('delete', 'delete', '', 'JTOOLBAR_DELETE', true);
```
Remove extension. Repeat in other instances.